### ISSUE TITLE
Why don't you update Kiwi anymore?

### ISSUE THREAD

**Issue by esbtc** [kiwibrowser/src.next#1195]
Why don't you update Kiwi anymore? It's really a pity that a browser this good is not developed anymore

**Comment by pokatomnik** [comment]
As It seems, this project is officially dead.
Consider using another browsers with extensions support (I will not recommend one, they can be easily found via Google).

**Comment by lje** [comment]
> Consider using another browsers with extensions support (I will not recommend one, they can be easily found via Google)

That list is quite short:
- Edge Canary
- Firefox and its forks Fennec and Mull

**Comment by pokatomnik** [comment]
> > Consider using another browsers with extensions support (I will not recommend one, they can be easily found via Google)
> 
> That list is quite short:
> 
>     * Edge Canary
> 
>     * Firefox and its forks Fennec and Mull

I tried Edge (Canary and stable), but I wasn't be able to install extensions from chrome web store.
There's only Firefox and its forks which are have a complete extensions support.
The another chromium-based one is Yandex browser (built with lots of internal advertisements)
So sad about it.

**Comment by lje** [comment]
> I tried Edge (Canary and stable), but I wasn't be able to install extensions from chrome web store.

You cannot (yet) install extensions directly from the store, it has to be done manually in the developer options (which need to be activated first by hitting the build number 5 times)

**Comment by pokatomnik** [comment]
> > I tried Edge (Canary and stable), but I wasn't be able to install extensions from chrome web store.
> 
> You cannot (yet) install extensions directly from the store, it has to be done manually in the developer options (which need to be activated first by hitting the build number 5 times)

Yep, I tried this approach. No such option listed. Maybe disabled for some regions, I guess.

**Comment by lje** [comment]
> Yep, I tried this approach. No such option listed. Maybe disabled for some regions, I guess.
![Edge Canary](https://github.com/user-attachments/assets/9fd99baf-2bb3-4158-9e52-1e89b6d4dbb7)

**Comment by lje** [comment]
I agree regarding "not a good choice", but unfortunately right now, it is the only Chromium based browser with full extension support, that is still being updated. Other browsers like Yandex only support a small fixed set of extensions.

**Comment by pokatomnik** [comment]
> I agree regarding "not a good choice", but unfortunately right now, it is the only Chromium based browser with full extension support, that is still being updated. Other browsers like Yandex only support a small fixed set of extensions.

I was using Yandex for a couple of months and was able to install almost any Manifest3 extension directly from Chrome web store.
The only thing I needed to do is to switch to desktop version.
Disclaimer: I am not giving advices to use Yandex, Edge or Firefox. Just personal experience.

**Comment by i11010520** [comment]
Is it possible to build kiwi/src.next without build.find.kiwi server?

**Comment by tathastu871** [comment]
Lemur had ported both devtools and extension from kiwi
extension sometime might be glithy laggy

But lemur had worst of all ui no tab grouping making browsing miserable

**Comment by tathastu871** [comment]
No tab groups
Mobile version of sites not optimized site contents are clipped had to repeatedly switch to desktop mode
Extensions open as panel not seperate tab just like lemur and thus the popup pages of extension not fully visible cannot make them fullscreen 
when you cant see the html completely because clipped from viewport its useless as a browser

**Comment by tathastu871** [comment]
its for sure their can never be one perfect browser with all features alteast basic features that makes a browser 'The Browser'

**Comment by tathastu871** [comment]
its for sure their can never be one perfect browser with all features alteast basic features that makes a browser 'The Browser'
Those new devs keep wasting time reimplementing UI, chromium has it covered you just need to do modification to make it more user friendly

Those brave browser devs keeps wasting their time in adding AI and crypto currency features they lack  simple  bookmark exporting feature

**Comment by lje** [comment]
Thanks, as I dont need tab groups, I checked Lemur and Quetta. Quetta seems pretty basic with almost no customization (it seems impossible to move search bar to top etc.).
Lemur looks good so far, I just can't find a way to disable live web search while typing something into the task bar. Lemur seems to have some kind of fixed "type ahead fing" which can't be disable.
Bookmark import from Kiwi works fine in both browsers.

**Comment by tathastu871** [comment]
> Thanks, as I dont need tab groups, I checked Lemur and Quetta. Quetta seems pretty basic with almost no customization (it seems impossible to move search bar to top etc.).
> Lemur looks good so far, I just can't find a way to disable live web search while typing something into the task bar. Lemur seems to have some kind of fixed "type ahead fing" which can't be disable.
> Bookmark import from Kiwi works fine in both browsers.

**Comment by tathastu871** [comment]
Lemur had been crashing after latest update so be careful when swithig

**Comment by pokatomnik** [comment]
> > > Consider using another browsers with extensions support (I will not recommend one, they can be easily found via Google)
> > 
> > 
> > That list is quite short:
> > 
> > * Edge Canary
> > * Firefox and its forks Fennec and Mull
> 
> Came across "Quetta Private Adblock Browser", supports addons from chrome extension store and MS edge addon store.
> 
> https://www.quetta.net/
> 
> But is only for mobile (android and ios and ipad). They say windows and mac OS is upcoming.

Not open source, unfortunately. Quetta is pretty new browser, so I am not sure It's trustworthy.
I'd rather trust a big tech company browser than a new one with closed sources. That's why I trusted Kiwi one day.

**Comment by tathastu871** [comment]
google chrome / chromium for android good and fast i had 1400 tabs open in both and yet no glitch or lag when switching tabs, yes to make switching fast they dont show preview if each tabs content thys it becones difficult to navigate


But But chromium for android latesr have addes search feature you can ditectky search open tabs using url or keywords and its amazing at first glance when i tried

As for ads in chromium or googlr chrome use adguard dns wide on your router or just adgurad app through vpn progile

**Comment by Reopado** [comment]
What happened to the update that was supposed to be dropping? That was weeks, maybe months ago now.  

It's a shame nobody with the skills can or will save Kiwi as it's by far the best browser for Android.

**Comment by Githubmaniak** [comment]
I wish someone would update Kiwi, Firefox for Android still has a lot of shortcomings.

**Comment by funnycups** [comment]
What a pity that Kiwi is the only browser offering support for add-ons, developer tools, and open-source features all in one. I hope it will continue to be developed.

**Comment by Iiridayn** [comment]
To answer the OP and others in thread, it is likely that Kiwi doesn't make enough money to afford to update it. People need to eat, and people with technical skills have many demands on their time and attention. It is not fair to demand releases unless you are a paying customer; and if you are, you vote with your feet.

While the developer or team might be facing a crisis unrelated to cashflow, the simple fact is, had they more money they could have more developers who could then pick up the slack when dealing with a crisis.

Please, do not be demanding, or sarcastic, or bitter, of whining about something you have gotten for free. Instead be grateful it has existed at all. If it's so valuable that you want to see it continue to exist, ask yourself how _you_ can help. Negative, critical, and rude feedback is a sure way to sap energy from developers - and without it, they are better served working for Microsoft, Google, or Amazon, etc.

Whining is not a way to help - it only drives away those who could.

**Comment by lje** [comment]
> Instead be grateful it has existed at all.

I agree - as with every free software, the developer does not owe continued support or development to the users.
There is still one thing I'd like: a clear statement that Kiwi development has ended and removal of the APKs from Github and Google Play. Providing a software with known security vulnerabilities that are actively being exploitet, puts unaware users at risk. Kiwi, in its current state, should not be available for download any more.

**Comment by pokatomnik** [comment]
> To answer the OP and others in thread, it is likely that Kiwi doesn't make enough money to afford to update it. People need to eat, and people with technical skills have many demands on their time and attention. It is not fair to demand releases unless you are a paying customer; and if you are, you vote with your feet.
> 
> While the developer or team might be facing a crisis unrelated to cashflow, the simple fact is, had they more money they could have more developers who could then pick up the slack when dealing with a crisis.
> 
> Please, do not be demanding, or sarcastic, or bitter, or whining about something you have gotten for free. Instead be grateful it has existed at all. If it's so valuable that you want to see it continue to exist, ask yourself how _you_ can help. Negative, critical, and rude feedback is a sure way to sap energy from developers - and without it, they are better served working for Microsoft, Google, or Amazon, etc.
> 
> Whining is not a way to help - it only drives away those who could.

Developers could help with implementing features and fixing bugs, but the browser is not open source at the moment. AFAIK, there is no donation button either. Therefore, the company that created the software has lost interest in supporting it.
My previous posts are just a warning about issues with outdated (and probably vulnerable) software.

**Comment by funnycups** [comment]
According to what I saw in Telegram groups, it seems that Kiwi is still under development but hasn't been released for performance or other reasons. Perhaps we could stay patient and eager for updates.

**Comment by lje** [comment]
> Perhaps we could stay patient and eager for updates.

Personal opinion: No, look for a new browser. Even if Kiwi gets an update today, it has been w/o security updates for too long. Leaving users at risk for such a long time is irresponsible. The currently available Kiwi version should have been removed months ago.

**Comment by tathastu871** [comment]
we are ready to pay few dollars,

**Comment by esbtc** [comment]
> we are ready to pay few dollars, we alreasy purchase sone good apps on play store why not this as donation
> 
> Problem is kiwi developer himself not responding he isnt replying to any of this threads or issues likely he already abondoned it he might not be login to his git account

Maybe he is dead and he is no longer in this world. I hope I am wrong

**Comment by gggirlgeek** [comment]
+1 for requesting updates. kiwi is my favorite on Android. On tablet, in laptop mode, it's the closest to desktop usage with addons. No Firefox variants allow selecting text to do a websearch (long click).

Here are the browsers I'm using. Almost all support addons. chromite is my default for apps opening sign in pages (too many tweaks and blockers in my everyday kiwi browser so I use "Better Open With" app to choose a browser depending on my needs.)

![image](https://github.com/user-attachments/assets/9142b15d-75b1-42ee-a91e-1343a072e2cd)

**Comment by gggirlgeek** [comment]
[Xbrowser](https://en.xbext.com/) is my second favorite chrome variant with extensions on Android.

**Comment by esbtc** [comment]
> [Xbrowser](https://en.xbext.com/) is my second favorite chrome variant with extensions on Android.

I installed cromite and XBrowser on Android but they don't support Chrome extensions

**Comment by chromer030** [comment]
[cromite](https://github.com/uazo/cromite) doesn't support extensions (yet).

**Comment by kakhulu31** [comment]
I don't want to sound like a shill, but it seems that currently the fastest chromium based android browser with tab grid support is Lemur (chromium 127), it's 40% faster than latest kiwi in speedometer. All the other ones are either slower than kiwi (quetta, mises) or gecko based (rainsee=kito browser) or don't have tab grid support (edge canary, yandex)

> XBrowser
It has a script manager preinstalled, a decent one, unlike what Via has, so you can use usercripts instead of extensions, for many people this is enough.

**Comment by esbtc** [comment]
how?
> 
> It has a script manager preinstalled, a decent one, unlike what Via has, so you can use usercripts instead of extensions, for many people this is enough.

**Comment by tathastu871** [comment]
lemur browser forked kiwi devtools and extension, they should have forked the whole kiwi and continued development

**Comment by funnycups** [comment]
> lemur browser forked kiwi devtools and extension, they should have forked the whole kiwi and continued development

It was however not open-sourced.

**Comment by Githubmaniak** [comment]
X Browser, Lemur are Chinese browsers. Therefore, there may be privacy issues. Quetta is also a company established through a Chinese virtual office rental company located in the UK, so there may be the same issues.

**Comment by MasterInQuestion** [comment]
"Well-control what data each layer may attain is the essence to the problem."
    On the East / West affairs.

    Though the privacy concerns of Chinese apps are serious:
    Prominently that from certain Shēn-Zhèn company.

**Comment by sankhababu** [comment]
FFupdater tagged Kiwi now as "no longer maintained" and "should no longer be used "

**Comment by tathastu871** [comment]
Will keep using till death do us apart (either one *,,*!!)
Cant live without DEVTools thats a essential requirement in current world where sites are bloated and broken, we need to test them and modify based on preferences

**Comment by serialpotato** [comment]
Extensions don't work on Lemur. Especially, bypasspaywall one. Lemur has pathetic extension support. Switch to Firefox instead. Edge has bloated Microsoft mess packed in

**Comment by kakhulu31** [comment]
> Extensions don't work on Lemur

Nah, there's no way they work worse on lemur, proxy extensions work better for example, on kiwi they were barely useable. 

>bypasspaywall

This is probably the issue, this extension is probably broken and outdated by now as it was dmca'ed

**Comment by tathastu871** [comment]
Kiwi is itself limited in extension support 
Extension apis like 
Omnibox, search api, tab groups, offline dom api etc etc are not implemented and some are there but disabled and not full fledged have many functions missing,

Even tabs api in kiwi is useless, most or tabs properties are immutable so tab api functions like grouping, moving, replace etc etc are not available only create update duplicate like primitive features only

**Comment by tathastu871** [comment]
Only good thing about kiwi chromium not being updates is you can keep using ublock like extensions,

If kiwi will update to latest chromium MV2 support will break and with MV3 only the adblocking extensions will be limited to 10% of their capabilities

**Comment by esbtc** [comment]
> Only good thing about kiwi chromium not being updates is you can keep using ublock like extensions,
> 
> If kiwi will update to latest chromium MV2 support will break and with MV3 only the adblocking extensions will be limited to 10% of their capabilities
> 
> Also extension that redirect resources and urls will break without webRequest API as its not in MV3

Yes, but I think in any case, an updated browser without an adblocker is much safer than an outdated browser with an adblocker.

**Comment by tathastu871** [comment]
> I used to use Kiwi an year ago. Then I switched to Cromite. Then I decided to stop using all dedicated chromium forks on Android. Then I started using [Privacy Browser](https://www.stoutner.com/privacy-browser-android/) which pulled me into Privacy rabbithole. And I started using NextDNS as a form of domain level adblocking, which works fine on all of my platform browsers. I don't think I need Extensions anymore. My screen on time reduced drastically too.

its not open source, it uses webview, so for normal browsing reading may be good 

but cant trust

**Comment by tathastu871** [comment]
Thats a dillema, no existing browsers on android or pc provides "Basic features that makes a browser 'A BROWSER' that being streamlined surfing experience


They have all sort of complex features like ai crypto etc but they lack good bookmarkling, backup sync fast tab opening and switching

**Comment by lje** [comment]
> Kiwi is a good browser. But like other ones, it's the same as any other chromium fork, but with extensions. That's it. Moreover, with the advent of MV3, content blocking via extensions is going to get restricted from March 2025 anyways. That makes ublock, adguard and other extensions almost obsolete.

No, that makes Chromium based browsers obsolete. What will remain are Brave with its "hardcoded" support for UBlock and Firefox based browsers like Fennec or Mull.

**Comment by lje** [comment]
At least in my bubble (which may represent a small minority) nobody would use a browser w/o proper adblock. Since we do not install random extensions, we consider the risk to get malware through an ad as significantly worse than getting malware through an extension.
Google doesn't remove V2 to protect users from malicious extensions but to kill ad-blockers - since they interfere with Google's business model. For this reason, Chrome for Android never supported extensions in the first place.

There may stillbe hope: apparently the US wants to force Google to split from Chrome. An independent Chrome may have diffenent priorities than shove ads on users' phones and screens, depending on who will fund it.

**Comment by tathastu871** [comment]
MV3 DNR MIGHT BE FASTER BUT 
DECLARATIVE NET REQUEST: 
1) IT HAS LIMITED REGEX FLAVOUR
2) IT IS STILL UNSTABLE, EVEN THE GOOGLE GITHUB SAMPLES FOR DNR ARE NOT WORKING
3) IT NOT ONLY BREAKS AD BLOCKERS, THERE ARE OTHER EXTENSION THAT INVOLVES REDIRECTION OF REQUESTS ALSO BREAK, ONE SUCH EXAMPLE IS "REDIRECTOR" EXTENSION
(AD BLOCKING - WE CAN COPE IP WITH USING ADGUARD BUT REDIRECTION BASED EXTENSIONS ARE ESSENTIAL
